package.json

```diff
@@ -24,11 +24,13 @@
     "typescript": "^3.1.6"
   },
   "scripts": {
-    "install": "webdriver-manager update",
+    "webdriver-update": "webdriver-manager update --gecko false --standalone false",
+    "install": "npm run webdriver-update || npm run webdriver-update",
```

Comment:
Not sure if there is a more concise way to say "try this twice if the first one fails" in this npm script context, open to suggestions to make this less weird, @djones @Robdel12 .

Reply:
Hmmmmmmmm, I don't really know of anything either. This maybe? https://www.npmjs.com/package/retry-cli But probably not worth it

Reply:
Yep, I saw that `retry-cli` but decided it was probably not worth it just to polish up this line.